### ISSUE TITLE
InfluxDB reporter

### ISSUE THREAD

**Issue by taybin** [Nextdoor/pg-bifrost#18]
After adding RabbitMQ support, I'd like to add an InfluxDB reporter, alongside Datadog.

**Comment by taybin** [comment]
Actually, I notice that the stats reporter is hardcoded to be datadog and isn't exposed in the CLI yet.

**Comment by stlava** [comment]
That is correct, I think the path forward on this is adding a `--reporter` option to the `replicate` command https://github.com/Nextdoor/pg-bifrost/blob/master/main/main.go#L479 and just have a switch statement to construct the appropriate reporter. The ticket is #3
